react-main/src/app/index.js

```diff
@@ -0,0 +1 @@
+export { Game } from './screens/Game';
```

Comment:
I would export this as App, since it will have its own logic soon. This component will be the core of the app